admin/base/settings/defaults.py

```diff
@@ -219,3 +224,10 @@
 
 ENTRY_POINTS = {'osf4m': 'osf4m', 'prereg_challenge_campaign': 'prereg',
                 'institution_campaign': 'institution'}
+
+
+# Change desk settings in local.py
+DESK_KEY = ''
+DESK_KEY_SECRET = ''
+DESK_TOKEN = ''
```

Comment:
My suggestion is to allow an admin user to enter their own tokens, enabling the Desk UI iff they have (possibly also requiring inclusion in a `desk_admin` 'group' or some such).